roboteam_ai/src/skills/Attack.cpp

```diff
@@ -46,6 +46,8 @@ bt::Node::Status Attack::onUpdate() {
 }
 
 void Attack::onTerminate(Status s) {
+    shot=false;
+    shotControl = std::make_shared<control::ShotController>(control::ShotPrecision::LOW ,control::BallSpeed::MAX_SPEED,false);
```

Comment:
shot is deprecated and the shotcontroller is set on initialize so this terminate is not necessary